src/types/transactions.types.ts

```diff
@@ -184,8 +184,9 @@ export interface GuardianScreenType extends WithClassnameType {
   title?: ReactNode;
   signStepInnerClasses?: SignStepInnerClassesType;
   signedTransactions?: DeviceSignedTransactions;
-  setSignedTransactions?: React.Dispatch<
-    React.SetStateAction<DeviceSignedTransactions | undefined>
+  guardianFormDescription?: ReactNode;
```

Comment:
guardianDescription / guardianSubtitle ?

Reply:
This is too generic, the description is strictly for the guardian input screen. 👍